.swiftlint.yml

```diff
@@ -23,3 +23,7 @@ trailing_comma:
   mandatory_comma: true
 type_body_length:
   warning: 400
+identifier_name:
+  max_length:
+    warning: 45
```

Comment:
This change is due to the identifier "clangPackageWithRelativePublicHeadersPath" exceeding the maximum allowed length. Its length is 42 characters.

Reply:
Default value of this is 40. The modified value is still lower than the default error value so should be okay.

https://realm.github.io/SwiftLint/identifier_name.html

<img width="684" height="695" alt="image" src="https://github.com/user-attachments/assets/9af4f912-1ce3-47fe-b96c-a47130859ba2" />

Reply:
I moved the rule to Tests/.swiftlint.yml in 8e95b49